### ISSUE TITLE
table('name', column(...)) does not work in alembic

### ISSUE THREAD

**Issue by AbdealiJK** [sqlalchemy/sqlalchemy2-stubs#161]
**Describe the bug**
In alembic, I define tables like:
```python
import sqlalchemy as sa
Book = sa.sql.table(
    'book',
    sa.sql.column('id', sa.Integer),
    sa.sql.column('name', sa.String),
)
```
But this fails saying:
```
error: Argument 2 to "TableClause" has incompatible type "ColumnClause[Integer]"; expected "Column[Any]"
error: Argument 2 to "TableClause" has incompatible type "ColumnClause[String]"; expected "Column[Any]"
```

**Expected behavior**
As per the documentation: https://docs.sqlalchemy.org/en/14/core/selectable.html#sqlalchemy.sql.expression.table
I see that the columns should be a list of `sqlalchemy.sql.expression.column` - but that does not seem to be what mypy expects

**To Reproduce**

```py
$ cat a.py
import sqlalchemy as sa
Book = sa.sql.table(
    'book',
    sa.sql.column('id', sa.Integer),
    sa.sql.column('name', sa.String),
)
```

**Error**
```
$ venv/bin/mypy --follow-imports=skip a.py
a.py:4: error: Argument 2 to "TableClause" has incompatible type "ColumnClause[Integer]"; expected "Column[Any]"
a.py:5: error: Argument 3 to "TableClause" has incompatible type "ColumnClause[String]"; expected "Column[Any]"
Found 2 errors in 1 file (checked 1 source file)
```

**Versions.**
 - OS: Ubuntu 18.04 on Windows 10 WSL2
 - Python: 3.7.4
 - SQLAlchemy: 1.4.22
 - mypy: 0.910
 - SQLAlchemy2-stubs: 0.0.2a11

**Additional context**
If I change this to:
```python
import sqlalchemy as sa
Book = sa.sql.table(
    'book',
    sa.Column('id', sa.Integer),
    sa.Column('name', sa.String),
)
```
it seems to work

**Comment by CaselIT** [comment]
thanks for the report

**Comment by CaselIT** [comment]
Released as v0.0.2a12
